docs/developer-tools/ThrowHelper.md

```diff
@@ -0,0 +1,153 @@
+---
+title: ThrowHelper
+author: Sergio0694
+description: Helper methods to efficiently throw exceptions
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, debug, net core, net standard
+dev_langs:
+  - csharp
+---
+
+# ThrowHelper
+
+The [ThrowHelper](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.diagnostics.ThrowHelper) class is a helper type that can be used to efficiently throw exceptions. It is meant to support the [Guard](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.diagnostics.guard) APIs, and it should primarily be used in situations where developers need fine-grained controls over the exception types being thrown, or over the exact exception message to include.
+
+## How it works
+
+The `ThrowHelper` class exposes a series of `ThrowException` APIs that offer a 1:1 mapping to the constructors of all the available exception types. These methods are meant to be used in place of the classic `throw new Exception(...)` statement in code, as it will result in smaller and more efficient code. Put simply:
```

Comment:
Worth mentioning (either directly or as the little purple box info note thingy) that it will result in a different stack trace

Reply:
Done in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/370/commits/c0e621ea1d90c47579e3218177ed3ab6f69656de.